README.rst

```diff
@@ -0,0 +1,191 @@
+.. This README is meant for consumption by humans and pypi. Pypi can render rst files so please do not use Sphinx features.
+   If you want to learn more about writing documentation, please check out: http://docs.plone.org/about/documentation_styleguide.html
+   This text does not appear on pypi or github. It is a comment.
+
+.. image:: https://travis-ci.org/collective/collective.xkey.svg?branch=master
+    :target: https://travis-ci.org/collective/collective.xkey
+
+.. image:: https://coveralls.io/repos/github/collective/collective.xkey/badge.svg?branch=master
+    :target: https://coveralls.io/github/collective/collective.xkey?branch=master
+    :alt: Coveralls
+
+.. image:: https://img.shields.io/pypi/v/collective.xkey.svg
+    :target: https://pypi.python.org/pypi/collective.xkey/
+    :alt: Latest Version
+
+.. image:: https://img.shields.io/pypi/status/collective.xkey.svg
+    :target: https://pypi.python.org/pypi/collective.xkey
+    :alt: Egg Status
+
+.. image:: https://img.shields.io/pypi/pyversions/collective.xkey.svg?style=plastic   :alt: Supported - Python Versions
+
+.. image:: https://img.shields.io/pypi/l/collective.xkey.svg
+    :target: https://pypi.python.org/pypi/collective.xkey/
+    :alt: License
+
+
+===============
+collective.xkey
+===============
+
+Use Varnish's `xkey module <https://github.com/varnish/varnish-modules>`_ for tag-based cache invalidation in Plone.
+
+This add-on is heavily inspired by and built upon on `collective.purgebyid <https://github.com/collective/collective.purgebyid>`_ (major credit goes to its contributors), but is more adapted to xkey's specific requirements.
```

Comment:
built upon on -> built upon
, but is more -> , but it is more

Reply:
Fixed